pkg/validation/errors/schema.go

```diff
@@ -483,7 +485,7 @@ func EnumFail(name, in string, value interface{}, values []interface{}) *Validat
 }
 
 // Required error for when a value is missing
-func Required(name, in string) *Validation {
+func Required(name, in string, value interface{}) *Validation {
```

Comment:
does this need the argument?

Reply:
It's not super important for `Required` but it might help differentiate `""` vs `nil` for example.

Reply:
For CRDs "" and null are both values that fulfill required

Reply:
Fixed and I also removed the unused `stringValidator` logic in https://github.com/kubernetes/kube-openapi/pull/213/commits/dc2906fe7cba3e29a77142b46d47f05c439aa781 as discussed in Slack.